contracts/src/v0.8/tests/FunctionsOracleWithInit.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Note: this is a wrapper contract for the go tests which don't have the Upgrades plugin
They _could_ call `initialize` themselves, but this makes it cleaner.